src/components/Navbar/navbar.css

```diff
@@ -1,4 +1,17 @@
 .nav-options:hover {
     background-color: #4b5563;
     transition: 0.2s ease-in;
+}
+
+.global-search::placeholder {
```

Comment:
**suggestion**: placeholders are not user-selectable so this will not affect them in any way.

Reply:
they are actually if you drag your cursor from outside the element